docker/docker-compose.ci.yml

```diff
@@ -73,6 +73,22 @@ services:
       PAYWALL_SCRIPT_URL: '${PAYWALL_SCRIPT_URL}'
       UNLOCK_STATIC_URL: '${UNLOCK_STATIC_URL}'
     command: ['npm', 'run', 'start']
+  paywall-integration:
+    image: paywall
+    ports:
+      - 3001:3001
+    expose:
+      - 3001
+    environment:
+      CI: '${CI}'
+      UNLOCK_ENV: '${UNLOCK_ENV}'
+      HTTP_PROVIDER: '${HTTP_PROVIDER}'
+      READ_ONLY_PROVIDER: '${READ_ONLY_PROVIDER}'
+      LOCKSMITH_URI: '${LOCKSMITH_URI}'
+      PAYWALL_URL: '${PAYWALL_URL}'
+      PAYWALL_SCRIPT_URL: '${PAYWALL_SCRIPT_URL}'
+      UNLOCK_STATIC_URL: '${UNLOCK_STATIC_URL}'
+    command: ['npm', 'run', 'integration-tests-start']
```

Comment:
here we call the `integration-tests-start` instead of `start` which triggers our build just-in-time with the correct env variables